app/assets/javascripts/govuk/selection-buttons.js

```diff
@@ -6,8 +6,6 @@
   if (typeof GOVUK === 'undefined') { root.GOVUK = {}; }
 
   var SelectionButtons = function (elmsOrSelector, opts) {
-    var $elms;
```

Comment:
Even if that's the case, I'm of the opinion that if it's in source control we should either:

- treat it as our responsibility and fix/maintain it
- declare it as an external dependency (using package.json in this situation)